src/rust/ide/view/src/project.rs

```diff
@@ -325,8 +361,8 @@ mod js {
 #[allow(missing_docs)]
 #[derive(Clone,CloneRef,Debug)]
 pub struct View {
-    model   : Model,
-    pub frp : Frp,
+    pub model : Model,
```

Comment:
making model pub is rather a bad practice. We should rather not do it. Why is it needed?

Reply:
`add_node_below` needs to be defined as a method on `Model`, such that we can easily use it from the FRP network. But we also need the method to demonstrate the auto-layout in the demo scene. The model needs to be public for that.